### ISSUE TITLE
Individual Location tabs list where else in the program the location is linked.

### ISSUE THREAD

**Issue by ryanreso1993** [Soyle-Productions/soyle-stories-desktop#70]
Given a location has been created
And given a character arc tab is open
When user links the location to a character arc sub-field

**Expected Result**
Then the location tab should be updated showing that it is linked to the character sub-field.

**Actual Result**
The location tab does not list the link.

**Notes**
Keeping track of where a location is linked in its Location details tab will help the writer to organize where the location is present or refered to while they write, for example, in characters arcs or symbols.

**Comment by ryanreso1993** [comment]
I believe what I wanted, to track the progress of character arcs in one location over different scenes, this can be accomplished now by opening the Location Details Tool alongside the Scene Characters tool of a character in that Scene which is listed in the Location Details tool. 

I think what I'm really vying for here is a more detailed Location Details tool, but it's not essential:

![image](https://user-images.githubusercontent.com/61167122/128271017-12827e93-6d42-465c-85c2-2742df20a081.png)

**Comment by b-camphart** [comment]
"more detailed Location Details tool," so, what do you want to see?

**Comment by ryanreso1993** [comment]
I believe something akin to this. I can see the description of the scene
and all the Scenes the Location hosts. But I’d like to expand the scenes
within the Location tool to see which characters are included and what part
of their arc is associated, as seen in the photo below. This would help me
as a writer track what happens in a location over scenes.

 This may also be accomplished by opening each Scene details listed for the
location. Having these details in the Location Details is just a shortcut,
a way to overview a location’s full role in a story. Perhaps I finish
writing LOTR and I see that the shire turned into shite by the end, and
looking at the overview of the arcs taking place there makes me realize I
want to balance the last scenes with the Shire being less shite.

Does that sense?

**Comment by b-camphart** [comment]
What would you expect to see in the shire?  The sections of Frodo's, Sam's, Gandolf's, Merry's, and Pippin's arcs in which their arcs started and then ended at the end of the story?  How does this help you balance the scenes?  What does balance the scenes mean to you?

**Comment by ryanreso1993** [comment]
Precisely. Don’t forget Bilbo’s too. Not their entire arcs mind you, but
the steps of the arc that appear in that scene in that location. Within the
Location Details Tool.

When I mean balance my story, I am looking at each element of my story and
seeing what ideas I can get rid of and what ideas I can continue exploring.

In this case, I want to examine my Locations. I ask myself, “What does each
Location in my story mean? What will readers remember when they remember
each location?” A Location is just a setting, so I want to see how I set my
scene. “What impression does this setting give as I begin reading it?” When
I picture the beginning of a scene, I imagine a frozen picture of a scene
before anything happens. Who is about to do what to who to get what.

I feel like seeing the characters and their arcs categorized by scene for
each location would help me achieve that imaginative exercise. The meaning
of a Location in my story comes from what is happening in each scene. Aka
the action done by characters in a step of their arc development.

Do this need sense more?

**Comment by ryanreso1993** [comment]
Questions for later:
"The next question is: what would you use that particular view for?  How does seeing all the characters, by scene, in the location give you deeper insight into what's going on?  What would you hope to gain from it?  It might be easier for me to understand if you use a concrete example instead of abstract ones.  So, instead of "I could balance the scene" it would be something like "I'd see Frodo and Sam in one scene in the Shire and in the final scene in the shire and so then I'd..."
August 8, 2021"
